app/test_split_integer.py

```diff
@@ -2,20 +2,23 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    assert sum(split_integer(17, 4)) == 17
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    min_num = min(split_integer(17, 4))
+    max_num = max(split_integer(17, 4))
+    difference = max_num - min_num
+    assert difference <= 1
```

Comment:
17 is not divisible by 4, we want to test if we split `number` in a number of parts that is a divisor of `number` - will we gut the list with equal parts?